Create TOBIT REGRESSIONS

duae-marriam opened the pull request.

duae-marriam committed:
Create TOBIT REGRESSIONS

DominikZabinski reviewed (commented):
Please add what libraries are needed and how you load the data.

duae-marriam committed:
Update TOBIT REGRESSIONS

Update to TOBIT REGRESSIONS.

Once reviewed, I can merge this code with the main analysis file

duae-marriam commented:
Added the part where required libraries are loaded and data is fetched.
Let me know if the rest is fine here

DominikZabinski commented on TOBIT REGRESSIONS:
```diff
@@ -1,4 +1,17 @@
+library(dplyr)
+library(tidyr)
+
+# loading data
+df <- read_csv("D:/4 Sem/RR/data_DMDDG.csv")
```
Could you please take a look at [this](https://github.com/DominikZabinski/rr_group_project/blob/main/supplementary.Rmd#L96-L102) part of code and change your so we can avoid reading files from local system and having hard-coded file paths? :)

duae-marriam commented:
Ohh, i see your point.
I am replacing that part of loading the data with this chunk

duae-marriam committed:
Update TOBIT REGRESSIONS

duae-marriam committed:
Update TOBIT REGRESSIONS
